docs/highlights/2020-10-27-kubernetes-integration.md

```diff
@@ -34,7 +32,7 @@ import Jump from '@site/src/components/Jump';
     across the board. This [further raises the bar][file_soure_benchmarks] in
     file tailing performance to meet the high demands of large-scale Kubernetes
     environments.
-2.  [**Adaptive Request Currency (ARC)**][adaptive_concurrency_post] -
+3.  [**Adaptive Request Currency (ARC)**][adaptive_concurrency_post] -
```

Comment:
For future reference, Markdown supports auto-inference of list numbers. Like this:

1. Apple
1. Orange
1. Banana

One Weird Trick!

Reply:
🤯 🤯 🤯